performance.md

```diff
@@ -1,162 +1,225 @@
+
+Remember also that there will be pages that should never be cached, as they are different for each user, such as the shopping cart page of an e-commerce, or the payment process and beyond.
+
+Another element to keep in mind when configuring caches is the usual time in which the information will be saved until it is invalidated. For example, a corporate website could cache the information for 1 week, a blog in which it is published infrequently, once a day would be enough but, on the other hand, an e-commerce website could cache it for only 1 hour.
 
-> In 2005, WordPress introduced its internal object cache — a way of automatically storing any data from the database (not just objects) in PHP memory to prevent unnecessary queries. However, out of the box, WordPress will discard all of those objects at the end of the request, requiring them to be rebuilt from scratch for the next page load.
+Some interesting plugins to manage the cache could be: [WP Super Cache](https://wordpress.org/plugins/wp-super-cache/), [WP Fastest Cache](https://wordpress.org/plugins/wp-fastest-cache/), [W3 Total Cache](https://wordpress.org/plugins/w3-total-cache/), [Simple Cache](https://wordpress.org/plugins/simple-cache/).
 
-###### source: [scalewp.io](https://www.scalewp.io/object-caching/)
 
-What does this mean? Think of a standard WordPress homepage displaying the most recent posts. Each of these posts has quite a bit of information associated with it WordPress must look up such as the author, categories, tags, and excerpt.
+In case you work with systems like nginx or LiteSpeed you have these: [LiteSpeed Cache](https://wordpress.org/plugins/litespeed-cache/), [Nginx Cache](https://wordpress.org/plugins/nginx-cache/).
 
-Support for a persistent object cache gives WordPress, plugins, and themes, a place to store that data for reuse. While these items are cached, PHP execution time is improved while lessening the load on the database. It's particularly helpful in situations where much of the page is difficult to cache from the front-end, like for authenticated traffic or e-commerce applications.
+### Opcode cache
 
-For these reasons, persistent object caching support is commonly offered with managed WordPress hosting.
+Every time a request comes to your web server, PHP has to execute and calculate everything, but PHP allows an internal operation caching system, which means that a copy of each execution is stored in memory or on disk. If it is active and the same operation is executed again, the system will take advantage of this system to return the calculation much faster, since it does not have to calculate everything completely.
 
-![](https://make.wordpress.org/hosting/files/2018/08/wordpress-object-caching-example.png)
+If your hosting provider has it active and configured, you can make use of it, but always taking into account that you will need a plugin to make the invalidation of the contents when necessary.
```

Comment:
I think it should be "may need", instead of "will need". Considering `opcache.validate_timestamp` may or may not be configured.

Reply:
updated